src/docs/output-targets/dist.md

```diff
@@ -32,38 +32,61 @@ Luckily, both builds can be generated at the same time, and shipped in the same
-|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-| `dir`    | The `dir` config specifies the public distribution directory. This directory is commonly the `dist` directory found within [npm packages](https://docs.npmjs.com/getting-started/packages). This directory is built and rebuilt directly from the source files. Additionally, since this is a build target, all files will be deleted and rebuilt after each build, so it's best to always copy source files into this directory. It's recommended that this directory not be committed to a repository. | `dist`  |
-| `empty`  | By default, before each build the `dir` directory will be emptied of all files. To prevent this directory from being emptied, change this value to `false`.                                                                                                                                                                                                                                                                                                                          | `true`  |
+### dir
 
+The `dir` config specifies the public distribution directory. This directory is commonly the `dist` directory found within [npm packages](https://docs.npmjs.com/getting-started/packages). This directory is built and rebuilt directly from the source files. Additionally, since this is a build target, all files will be deleted and rebuilt after each build, so it's best to always copy source files into this directory. It's recommended that this directory not be committed to a repository.
+
+This option defaults to `dist` when omitted from a Stencil configuration file.
+
+### empty
+
+By default, before each build the `dir` directory will be emptied of all files. To prevent this directory from being emptied, change this value to `false`.
+
+This flag defaults to `true` when omitted from a Stencil configuration file.
+
+### transformAliasedImportPathsInCollection
+
+This option will allow [path aliases](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) defined in a project's `tsconfig.json` to be transformed into relative paths in the output code for the implicit `dist-collection` output target. This does not affect imports for external packages.
```

Comment:
My only concern here is that unless you work for Ionic/hack on Stencil, "implicit `dist-collection` output target" is a little confusing. Maybe the better thing to do is to rephrase this in terms on the output of the output target, in a way that's more relatable to an end user?
```suggestion
This option will allow [path aliases](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) defined in a project's `tsconfig.json` to be transformed into relative paths in the code output under the `dist/collection` subdirectory for this output target. This does not affect imports for external packages.
```

Reply:
Good thought. I made that change, but also added the `collectionDir` option to the docs and linked that with what you were saying.